rastervision/data/label_store/semantic_segmentation_raster_store_config.py

```diff
@@ -64,27 +98,71 @@ def update_for_command(self,
                 msg = 'No URI set for SemanticSegmentationRasterStoreConfig'
                 io_def.add_missing(msg)
 
+            for vo in self.vector_output:
+                io_def.add_input(vo['uri'])
+
         return io_def
 
 
 class SemanticSegmentationRasterStoreConfigBuilder(LabelStoreConfigBuilder):
     def __init__(self, prev=None):
         config = {}
         if prev:
-            config = {'uri': prev.uri, 'rgb': prev.rgb}
+            config = {
+                'uri': prev.uri,
+                'vector_output': prev.vector_output,
+                'rgb': prev.rgb,
+            }
 
         super().__init__(SemanticSegmentationRasterStoreConfig, config)
 
     def from_proto(self, msg):
-        return self.with_uri(msg.semantic_segmentation_raster_store.uri) \
-                   .with_rgb(msg.semantic_segmentation_raster_store.rgb)
+        uri = msg.semantic_segmentation_raster_store.uri
+        rgb = msg.semantic_segmentation_raster_store.rgb
+        vo = msg.semantic_segmentation_raster_store.vector_output
+
+        return self.with_uri(uri) \
+                   .with_vector_output(vo) \
+                   .with_rgb(rgb)
 
     def with_uri(self, uri):
         """Set URI for a GeoTIFF used to read/write predictions."""
         b = deepcopy(self)
         b.config['uri'] = uri
         return b
 
+    def with_vector_output(self, vector_output):
+        """Configure vector output for predictions.
+
+            Args:
+                vector_output: Either a list of dictionaries or a
+                    protobuf object.  The dictionary or the object
+                    contain (respectively) keys (attributes) called
+                    'uri', 'class_id', and 'mode.  The 'uri' key is
+                    either a file where the GeoJSON prediction will be
+                    written, or "" indicating that the filename should
+                    be auto-generated.  'class_id' is the integer
+                    prediction class that is of interest.  The 'mode'
+                    key must be set to 'buildings' or 'polygons'.
+
+        """
+        b = deepcopy(self)
+        ar = []
+
+        if isinstance(vector_output, list):
+            for vo in vector_output:
+                ar.append(vo.copy())
+        else:
+            for vo in vector_output:
+                ar.append({
+                    'uri': vo.uri,
+                    'mode': vo.mode,
+                    'class_id': vo.class_id
```

Comment:
This produces a separate file for each class/mode. That has the advantage of allowing you to use several different modes for each class. I could see how that could be useful. On the other hand, it produces a file(s) for each class, rather than a single file where each feature has a `class_id` field, which is how things work in other parts of RV. I think it's ok the way it is, but I just wanted to note this tradeoff.